assets/js/modules/analytics/index.legacy.js

```diff
@@ -25,56 +25,12 @@ import { addFilter } from '@wordpress/hooks';
  * Internal dependencies
  */
 import { getModulesData } from '../../util';
-import { createAddToFilter } from '../../util/helpers';
```

Comment:
We can also remove the following:

* The `countriesByTimeZone` constant in the `assets/js/modules/analytics/util/countries-timezones.js` file
* The `translateAnalyticsError` function in the `assets/js/modules/analytics/util/index.js` file